src/activities/parse-docmap.ts

```diff
@@ -3,9 +3,15 @@ import { parser } from '@elifesciences/docmap-ts';
 export const parseDocMap = async (docMapInput: string): Promise<parser.ManuscriptData> => {
   const manuscriptData = parser.parsePreprintDocMap(docMapInput);
 
-  if (!manuscriptData) {
+  try {
+    // TO-DO: This is a temporary hack, remove once DocMaps are fixed
+    // eslint-disable-next-line @typescript-eslint/dot-notation
+    const tdmPath: string = JSON.parse(docMapInput)['steps']['_:b0']['actions'][0]['outputs'][0]['_tdmPath'];
+    if (manuscriptData?.versions?.[0]?.preprint) manuscriptData.versions[0].preprint.content = tdmPath;
+    else throw Error();
```

Comment:
I'm not sure we want to error here - this is making the assumption the first step has a _tdmPath, and that it should replace version 0's preprint content. The original condition - if manuscriptData is empty we should error - stands as a valid condition to respond to, in additional to the new logic we want.

On the logic itself - I'm not sure it's going to be forward compatible because if the docmap is fixed, we'll replace a valid preprint content URL with an invalid/undefined. 

It also not taking into account revised preprint that isn't step `_:b0` (see step `_:b3` of https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=85111)

Instead, we should probably be looping all steps, all inputs, and constructing an additional content entry to inject into the docmap, then handing the "fixed" docmap to the parser. Happy to explain more if that's not making any sense.